packages/variants/src/variants.ts

```diff
@@ -170,7 +188,13 @@ export function getSoftVariant(theme: IPartialTheme): ITheme {
     // inputFocusBorderAlt: p.themePrimary,
     variantBorder: !fullTheme.isInverted ? p.neutralLight : p.neutralLighterAlt,
     variantBorderHovered: p.neutralTertiary,
-    defaultStateBackground: !fullTheme.isInverted ? p.themeLight : p.themeTertiary,
+    defaultStateBackground: !fullTheme.isInverted ? p.neutralLight : p.neutralQuaternaryAlt,
+
+    actionLink: p.neutralPrimary,
+    actionLinkHovered: !fullTheme.isInverted ? p.neutralDark : p.neutralPrimary,
```

Comment:
is this right? in inverted, the color doesn't change on hover?

Reply:
@phkuo Per the designer, I'm going to wait on this until a few things can get resolved on the design end. Apparently there is a lot of back and forth right now on link interactions. I'll close this PR and reopen once I have confirmation.